doc/api/test.md

```diff
@@ -1265,6 +1268,9 @@ changes:
   * `forceExit`: {boolean} Configures the test runner to exit the process once
     all known tests have finished executing even if the event loop would
     otherwise remain active. **Default:** `false`.
+  * `globPatterns`: {Array} An array containing the list of glob patterns to
```

Comment:
Can you mention that this cannot be used with the `files` option.

Reply:
done